utils/node_local_test.py

```diff
@@ -2699,7 +2685,7 @@ def test_cont_clone(self):
         # the container.  Check the returncode only, do not verify the data.
         # tempfile() will remove the directory on completion.
         src_dir = tempfile.TemporaryDirectory(prefix='copy_src_',)
-        ofd = open(os.path.join(src_dir.name, 'file'), 'w')
+        ofd = open(join(src_dir.name, 'file'), 'w')
```

Comment:
(pylint-consider-using-with) Consider using 'with' for resource-allocating operations